css/main.css

```diff
@@ -0,0 +1,653 @@
+}
+
+.project-stacks {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 15px;
+  padding: 20px 0;
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.project-stacks li:hover {
+  background: rgba(0, 0, 0, 0.349);
+}
+
+#about-me,
+.about-me-section,
+.stacks {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  border: 1px solid transparent;
+  transition: 350ms;
+  cursor: pointer;
+  outline: none;
+}
+
+.link:hover {
+  color: #f55800;
+  background: #fff5e1;
+  border: 1px solid #f55800;
+}
+
+.centered-link {
+  align-self: center;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url("../media/svg/rectangle.svg");
+}
+
+.frameworks {
+  list-style: inside url("../media/svg/square.svg");
+}
+
+.skills {
+  list-style: inside url("../media/svg/circle.svg");
+}
+
+#contact {
+  padding: 10%;
+}
+
+#contact h2 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+input {
+  border: none;
+  outline: none;
+  border-bottom: solid 2px #dbd8d7;
+}
+
+textarea {
+  color: #3c3a39;
+  border: none;
+  outline: none;
+}
+
+input,
+textarea {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  margin: 5px;
+  padding: 5px;
+  min-width: 50vw;
+}
+
+.name-form {
+  display: none;
+}
+
+.footer-social {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding: 5vh;
+  filter:
+    invert(67%) sepia(46%) saturate(986%) hue-rotate(179deg)
+    brightness(92%) contrast(82%);
+}
+
+/* styling of my modal ------------------------------------------ */
+.nav_modal-background {
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  z-index: 6;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  backdrop-filter: blur(4px);
+  visibility: hidden;
+  background-color: transparent;
+}
```

Comment:
- [ ] Please stick to the design as much as possible.

- Your menu design is like this  
![Screenshot from 2022-04-13 21-51-00](https://user-images.githubusercontent.com/33752735/163260968-092fb76f-d3ad-410f-8415-c11fe282568e.png)

- But it should be like this 
![Screenshot from 2022-04-13 21-52-09](https://user-images.githubusercontent.com/33752735/163261062-0fff3999-5446-45fe-a03e-b44f8e45f5ca.png)